src/test/resources/io/jenkins/plugins/gitlabserver/casc/configuration-as-code.yml

```diff
@@ -0,0 +1,16 @@
+credentials:
```

Comment:
I am thinking adding a HashTable or HashMap should not create any incompatibility issues. As it will never be configured. They are only for internal purpose for faster GitLab servers fetch. And everytime List of servers are configured, the Map will also get repopulated. Will it still cause incompatibility?

Reply:
Or maybe I can try implementing it and see if it works. I am doing that.

Reply:
A HashMap addition to `GitLabServers` worked with JCasC as long as it didn't get instantiated in the constructor. But we removed it as it didn't seem like the most quintessential thing to do. So we stick to the default implementation of a list of servers.